src/frontend/Dockerfile.dev

```diff
@@ -1,7 +1,9 @@
-FROM node:12.16-alpine
+FROM node:13.14-alpine
 
 WORKDIR /app
 
 RUN apk add --update --no-cache git bash yarn nano
 
+RUN apk add --update --no-cache autoconf libtool automake alpine-sdk
```

Comment:
Je fusionnerai les deux lignes

Reply:
en fait je ne sais pas vraiment à quoi sert la première...

Reply:
Ca permet d'utiliser certaines commandes dans le container (en fait `docker exec -it`)

Reply:
ok, je comprends mieux. c'est une pratique que je fais tres peu. du coup je les garderai bien séparer pour distinguer les outil de "développement" des dépendance système nécessaire pour builder.